<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParaSet</class>
 <widget class="QDialog" name="ParaSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>连接参数设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>设备通道数：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3" colspan="2">
           <widget class="QListWidget" name="list"/>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QLabel" name="buletoothState">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>请选择蓝牙设备：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3" colspan="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>串口号：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item row="0" column="4">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>波特率：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>数据位：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>校验位：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="search">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>停止位：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>1Channel</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4Channel</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>52Channel</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
